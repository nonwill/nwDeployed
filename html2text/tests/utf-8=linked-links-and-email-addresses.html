<div dir="ltr"><div class="gmail_default" style="font-family:arial,helvetica,sans-serif">Supersedes: <a href="https://archives.gentoo.org/gentoo-dev/message/06b4c45858a15e78cb44efd6cf679600" style="font-family:Arial,Helvetica,sans-serif">https://archives.gentoo.org/gentoo-dev/message/06b4c45858a15e78cb44efd6cf679600</a></div><div class="gmail_default" style="font-family:arial,helvetica,sans-serif"><br></div><div class="gmail_default" style="font-family:arial,helvetica,sans-serif"><span style="font-family:Arial,Helvetica,sans-serif">I would like to reserve </span><span class="gmail-il" style="font-family:Arial,Helvetica,sans-serif">GID</span><span style="font-family:Arial,Helvetica,sans-serif"> 407 for games-util/</span><span class="gmail-il" style="font-family:Arial,Helvetica,sans-serif">gamemode</span><span style="font-family:Arial,Helvetica,sans-serif">.</span><br style="font-family:Arial,Helvetica,sans-serif"><br style="font-family:Arial,Helvetica,sans-serif"><span style="font-family:Arial,Helvetica,sans-serif">As far as I can tell, </span><span class="gmail-il" style="font-family:Arial,Helvetica,sans-serif">GID</span><span style="font-family:Arial,Helvetica,sans-serif"> 407 is free [1]</span><br style="font-family:Arial,Helvetica,sans-serif"><br style="font-family:Arial,Helvetica,sans-serif"><span style="font-family:Arial,Helvetica,sans-serif">Here&#39;s a PR for this change [2]</span><br style="font-family:Arial,Helvetica,sans-serif"><br style="font-family:Arial,Helvetica,sans-serif"><span style="font-family:Arial,Helvetica,sans-serif">[1] </span><a href="https://api.gentoo.org/uid-gid.txt" rel="noreferrer" target="_blank" style="font-family:Arial,Helvetica,sans-serif">https://api.gentoo.org/uid-<span class="gmail-il">gid</span>.txt</a><br style="font-family:Arial,Helvetica,sans-serif"><span style="font-family:Arial,Helvetica,sans-serif">[2] </span><a href="https://github.com/gentoo/gentoo/pull/13158" rel="noreferrer" target="_blank" style="font-family:Arial,Helvetica,sans-serif">https://github.com/gentoo/gentoo/pull/13158</a><br></div></div><br><div class="gmail_quote"><div dir="ltr" class="gmail_attr">Am Mi., 16. Okt. 2019 um 21:12 Uhr schrieb K K &lt;<a href="mailto:xxxxxxxx77%2Bbgo@xxxxx.com">xxxxxxxx77+bgo@xxxxx.com</a>&gt;:<br></div><blockquote class="gmail_quote" style="margin:0px 0px 0px 0.8ex;border-left:1px solid rgb(204,204,204);padding-left:1ex">Resent because I used the wrong &quot;From:&quot;...<br>
<br>
Am Mo., 14. Okt. 2019 um 10:13 Uhr schrieb K K &lt;<a href="mailto:xxxxxxxx77@xxxxx.com" target="_blank">xxxxxxxx77@xxxxx.com</a>&gt;:<br>
&gt;<br>
&gt; I would like to reserve GID 405 for games-util/gamemode.<br>
&gt;<br>
&gt; As far as I can tell, GID 405 is free [1]<br>
&gt;<br>
&gt; Here&#39;s a PR for this change [2]<br>
&gt;<br>
&gt; [1] <a href="https://api.gentoo.org/uid-gid.txt" rel="noreferrer" target="_blank">https://api.gentoo.org/uid-gid.txt</a><br>
&gt; [2] <a href="https://github.com/gentoo/gentoo/pull/13158" rel="noreferrer" target="_blank">https://github.com/gentoo/gentoo/pull/13158</a><br>
</blockquote></div>
