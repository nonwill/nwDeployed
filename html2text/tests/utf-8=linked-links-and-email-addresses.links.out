Supersedes: _h_t_t_p_s_:_/_/_a_r_c_h_i_v_e_s_._g_e_n_t_o_o_._o_r_g_/_g_e_n_t_o_o_-_d_e_v_/_m_e_s_s_a_g_e_/
_0_6_b_4_c_4_5_8_5_8_a_1_5_e_7_8_c_b_4_4_e_f_d_6_c_f_6_7_9_6_0_0[1]

I would like to reserve GID 407 for games-util/gamemode.

As far as I can tell, GID 407 is free [1]

Here's a PR for this change [2]

[1] _h_t_t_p_s_:_/_/_a_p_i_._g_e_n_t_o_o_._o_r_g_/_u_i_d_-_g_i_d_._t_x_t[2]
[2] _h_t_t_p_s_:_/_/_g_i_t_h_u_b_._c_o_m_/_g_e_n_t_o_o_/_g_e_n_t_o_o_/_p_u_l_l_/_1_3_1_5_8[3]

Am Mi., 16. Okt. 2019 um 21:12 Uhr schrieb K K <_x_x_x_x_x_x_x_x_7_7_+_b_g_o_@_x_x_x_x_x_._c_o_m[4]>:
> Resent because I used the wrong "From:"...
>
> Am Mo., 14. Okt. 2019 um 10:13 Uhr schrieb K K <_x_x_x_x_x_x_x_x_7_7_@_x_x_x_x_x_._c_o_m[5]>:
> >
> > I would like to reserve GID 405 for games-util/gamemode.
> >
> > As far as I can tell, GID 405 is free [1]
> >
> > Here's a PR for this change [2]
> >
> > [1] _h_t_t_p_s_:_/_/_a_p_i_._g_e_n_t_o_o_._o_r_g_/_u_i_d_-_g_i_d_._t_x_t[6]
> > [2] _h_t_t_p_s_:_/_/_g_i_t_h_u_b_._c_o_m_/_g_e_n_t_o_o_/_g_e_n_t_o_o_/_p_u_l_l_/_1_3_1_5_8[7]


############ RReeffeerreenncceess ############
   1. https://archives.gentoo.org/gentoo-dev/message/
      06b4c45858a15e78cb44efd6cf679600
   2. https://api.gentoo.org/uid-gid.txt
   3. https://github.com/gentoo/gentoo/pull/13158
   4. mailto:xxxxxxxx77%2Bbgo@xxxxx.com
   5. mailto:xxxxxxxx77@xxxxx.com
   6. https://api.gentoo.org/uid-gid.txt
   7. https://github.com/gentoo/gentoo/pull/13158
